apps/crn-server/test/data-providers/calendars.data-provider.test.ts

```diff
@@ -172,7 +172,12 @@ describe('Calendars data provider', () => {
           order: 'data/name/iv asc',
         },
       );
-      expect(result).toEqual({ total: 1, items: [getCalendarDataObject()] });
+      expect(result).toEqual({
+        total: 1,
+        items: [
+          { ...getCalendarDataObject(), workingGroup: true, group: true },
```

Comment:
why not update the fixture? It's strange you are having to do this here and not in here https://github.com/yldio/asap-hub/blob/2471f1141b14d49b3ebd2b1aeec95cdf59e994d3/apps/crn-server/test/data-providers/calendars.data-provider.test.ts#L99